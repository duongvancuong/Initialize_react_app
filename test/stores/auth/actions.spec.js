import {
  authenticateUser,
  authenticateUserSuccess,
  authenticateUserError,
  loadingUserData,
  logoutUser,
  logoutUserSuccess,
  cleanErrorLogin,
  cleanErrorLoginSuccess,
  handleExceptionUserError,
} from '../../../src/stores/auth/actions';

describe('Actions', () => {
  it('authenticateUser should return an action', () => {
    expect(authenticateUser()).toMatchSnapshot();
  });
  it('authenticateUserSuccess should return an action', () => {
    expect(authenticateUserSuccess()).toMatchSnapshot();
  });
  it('authenticateUserError should return an action', () => {
    expect(authenticateUserError()).toMatchSnapshot();
  });
  it('loadingUserData should return an action', () => {
    expect(loadingUserData()).toMatchSnapshot();
  });
  it('logoutUser should return an action', () => {
    expect(logoutUser()).toMatchSnapshot();
  });
  it('logoutUserSuccess should return an action', () => {
    expect(logoutUserSuccess()).toMatchSnapshot();
  });
  it('cleanErrorLogin should return an action', () => {
    expect(cleanErrorLogin()).toMatchSnapshot();
  });
  it('cleanErrorLoginSuccess should return an action', () => {
    expect(cleanErrorLoginSuccess()).toMatchSnapshot();
  });
  it('handleExceptionUserError should return an action', () => {
    expect(handleExceptionUserError()).toMatchSnapshot();
  });
});
