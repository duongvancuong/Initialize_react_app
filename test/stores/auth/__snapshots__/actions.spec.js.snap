// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Actions authenticateUser should return an action 1`] = `
Object {
  "type": "AUTHENTICATE_USER",
}
`;

exports[`Actions authenticateUserError should return an action 1`] = `
Object {
  "type": "AUTHENTICATE_USER_ERROR",
}
`;

exports[`Actions authenticateUserSuccess should return an action 1`] = `
Object {
  "type": "AUTHENTICATE_USER_SUCCESS",
}
`;

exports[`Actions cleanErrorLogin should return an action 1`] = `
Object {
  "payload": Object {},
  "type": "CLEAN_ERROR_LOGIN",
}
`;

exports[`Actions cleanErrorLoginSuccess should return an action 1`] = `
Object {
  "type": "CLEAN_ERROR_LOGIN_SUCCESS",
}
`;

exports[`Actions handleExceptionUserError should return an action 1`] = `
Object {
  "payload": Object {},
  "type": "HANDLE_EXCEPTION_USER_ERROR",
}
`;

exports[`Actions loadingUserData should return an action 1`] = `
Object {
  "type": "LOADING_USER_DATA",
}
`;

exports[`Actions logoutUser should return an action 1`] = `
Object {
  "payload": Object {},
  "type": "LOGOUT_USER",
}
`;

exports[`Actions logoutUserSuccess should return an action 1`] = `
Object {
  "type": "LOGOUT_USER_SUCCESS",
}
`;
